src/pages/api/manifest.ts

```diff
@@ -13,11 +16,9 @@ export default createManifestHandler({
       id: "saleor.app",
```

Comment:
@przlada can you check how ID is consumed in the Core? 

@peelar probably you should set your own ID